### PR TITLE
[ONNX] Update Split operator in unsafe_chunk() method.

### PR DESCRIPTION
**Pull request by fatcat-z**
Update unsafe_chunk() method for opset 13 Split.

### CODE REVIEW

File: test/onnx/test_pytorch_onnx_onnxruntime.py [hunk 1]
```diff
@@ -6200,6 +6200,26 @@ def forward(self, images, features):
                       test_with_inputs=[(images, features), (images2, test_features)],
                       dict_check=False)
 
+    @skipIfUnsupportedMinOpsetVersion(11)
+    @disableScriptTest()
+    def test_unsafe_chunk(self):
+        class ChunkModel(torch.nn.Module):
+            def __init__(self):
```
Comment (KsenijaS) [inline test/onnx/test_pytorch_onnx_onnxruntime.py:6207]: why is init is needed here?

File: torch/onnx/symbolic_opset13.py [hunk 2]
```diff
@@ -142,3 +142,26 @@ def reduce_dim(g, self, dim, keepdim, dtype):
...
+
+@parse_args('v', 'i', 'i', 'i')
+def unsafe_chunk(g, self, chunks, dim, _outputs=None):
+    if _outputs is None:
+        return g.op("SplitToSequence",
+                    self,
+                    g.op("Constant", value_t=torch.tensor(1, dtype=torch.long)),
+                    axis_i=dim, keepdims_i=0)
+
+    size = sym_help._get_tensor_dim_size(self, dim)
```
Comment (BowenBao) [inline torch/onnx/symbolic_opset13.py:154]: could `size` potentially be a symbol?

Comment (neginraoof) [reply torch/onnx/symbolic_opset13.py:154]: This part could be implemented more dynamically, but we were looking at the conditional logic in line 160. The dynamic implementation of this part was more complex in this case.

File: test/onnx/test_pytorch_onnx_onnxruntime.py (hunk 1 above)
Comment (fatcat-z) [reply test/onnx/test_pytorch_onnx_onnxruntime.py]: Good catch. It is not necessary.

**Review by neginraoof — APPROVED** [review]
LGTM. Thanks.

**Comment by neginraoof** [comment]
Please rebase this before merge.

**Pull request closed without merging**